<?xml version="1.0"?>
<launch>

  <node name="tool_surface_calib_control_node" pkg="dumbo_tool_surface_calib" type="tool_surface_static_calib_control_node" cwd="node" respawn="false" output="screen" >
    <rosparam command="load" file="$(find dumbo_tool_surface_calib)/config/tool_surface_calib_control_static.yaml"/>
    <!-- <remap from="/left_arm_FT_sensor/FT_compensated" to="/left_arm_FT_sensor/FT_raw"/> -->
  </node>

  <include file="$(find dumbo_safety_controller)/launch/left_arm_safety_controller.launch"/>

  <!-- <include file="$(find dumbo_door_opening)/launch/ar_kinect.launch"/> -->

  <!-- <include file="$(find dumbo_arm_navigation)/launch/dumbo_arm_navigation.launch"/> -->

</launch>